app/models/manageiq/providers/lenovo/physical_infra_manager/parsers/parser_dictionary_constants.rb

```diff
@@ -37,6 +37,29 @@ class ParserDictionaryConstants
     # and your values (string) corresponds to the attributes of the
     # source object who the value will be set as value of the key of result hash.
     # see +ManageIQ::Providers::Lenovo::Parse#parse+
+    SWITCH = {
+      :name          => 'name',
+      :uid_ems       => 'uuid',
+      :switch_uuid   => 'uuid',
+      :health_state  => 'overallHealthState',
+      :power_state   => 'powerState',
+      :product_name  => 'productName',
+      :part_number   => 'partNumber',
+      :serial_number => 'serialNumber',
+      :description   => 'description',
+      :manufacturer  => 'manufacturer',
+    }.freeze
+
+    SWITCH_PORT = {
+      :device_name => 'portName',
+      :address     => 'peerMacAddress',
```

Comment:
The _peerMacAddress_ field being returned by LXCA 2.0.0 seems to be empty for all ports. As a result, the switch ports in my _guest_device_ table have an empty _address_ field. Is the _peerMacAddress_ field normally populated?

Reply:
That is a good question. The last LXCA 2.x instances I had access do not have that data anymore. I think this is particularly important because it is the way we have for acquiring the mac address. I will further investigate.

Reply:
@skovic, I started looking for that, but I am convinced that this is the field we want here, despite that points to the address of the entity attached to the port. There is no data in the LXCA API to fill the address in the ManageIQ. You and @rodneyhbrown7 have more experience with the LXCA platform, so if you think we should change that to another field, let me know. Anyway, I think that if we do not have the proper information here, there is a problem in how the LXCA acquires the data.

Reply:
Since the model we are creating in the database represents the switch and its ports, I don't think it is correct to use the _peerMacAddress_ as the address of the switch's port because it is the MAC address of the device that's attached to the switch -- not the MAC address of the switch port itself. @rodneyhbrown7 Should we not fill in this field, or can we somehow get the switch port MAC addresses from LXCA?

Reply:
@skovic, @rodneyhbrown7, as you said in our meeting, I will create a new `peerMacAddress` field in the schema to store that data and I will leave the address blank.